src/CronTask.php

```diff
@@ -1959,18 +1959,13 @@ public static function showSearchStatusArea()
         if (count($warnings) > 0) {
             $msg = __('Automatic actions may not be running as expected');
```

Comment:
A `|raw` filter is used in the template, HTML special chars have to be escaped here.

```suggestion
            $msg = __s('Automatic actions may not be running as expected');
```